spk/bazarr/Makefile

```diff
@@ -1,26 +1,37 @@
 SPK_NAME = bazarr
 
-SPK_VERS = 1.0.1
-SPK_REV = 2
+SPK_VERS = 1.0.2
+SPK_REV = 3
 
 SPK_ICON = src/bazarr.png
 
-BUILD_DEPENDS = cross/python38
+BUILD_DEPENDS = cross/python310
 
 DEPENDS = cross/$(SPK_NAME)
 DEPENDS += cross/unrar
-DEPENDS += cross/numpy
-DEPENDS += cross/webrtcvad
-DEPENDS += cross/greenlet
 
-SPK_DEPENDS = "python38:ffmpeg"
+SPK_DEPENDS = "python310:ffmpeg"
+
+# [numpy] gcc-4.6.4 failed with exit status 1
+# https://github.com/numpy/numpy/issues/20664
+UNSUPPORTED_ARCHS = $(ARMv5_ARCHS)
+
+include ../../mk/spksrc.common.mk
+
+# [numpy] Fix wheel building with older compilers
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+WHEELS_CPPFLAGS = [numpy] -std=c++0x
+ifeq ($(findstring $(ARCH),$(ARMv8_ARCHS)),$(ARCH))
+WHEELS_CFLAGS = [numpy] -O0
+endif
+endif
```

Comment:
Open question: Shouldn't this section be moved before the inclusion of `include ../../mk/spksrc.spk.mk` ?

Reply:
it is, it requires the `version_lt` function so it has `include ../../mk/spksrc.common.mk` above.. then further down below you find the `include ../../mk/spksrc.spk.mk`

Reply:
I meant it could be closer to that line, right ?

Reply:
yup it can, now moved.